ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/ClustersCoverageTable.tsx

```diff
@@ -26,26 +26,43 @@ import {
     complianceEnhancedCoverageClustersPath,
     complianceEnhancedScanConfigsPath,
 } from 'routePaths';
-import { getAllClustersCombinedStats } from 'services/ComplianceEnhancedService';
+import {
+    getAllClustersCombinedStats,
+    getAllClustersCombinedStatsCount,
+} from 'services/ComplianceEnhancedService';
+import { SortOption } from 'types/table';
 
+import useURLSort from 'hooks/useURLSort';
 import ComplianceClusterStatus from '../ScanConfigs/components/ComplianceClusterStatus';
 import CoverageTableViewToggleGroup from './Components/CoverageTableViewToggleGroup';
-
 import {
     calculateCompliancePercentage,
     getCompliancePfClassName,
     getStatusCounts,
 } from './compliance.coverage.utils';
 
+const sortFields = ['Cluster'];
+const defaultSortOption = {
+    field: 'Cluster',
+    direction: 'asc',
+} as SortOption;
```

Comment:
Observation, not change request: `const defaultSortOption: SortOption = {` is an alternative.

TypeScript will report an error if properties and values are not consistent with declared type, but you can confirm by temporary local experiment whether `as` (which I read as trust me) might allow some errors?

Reply:
I actually didn't know that 😯 

I imagine I pulled that from somewhere else in our code base but I can't think of an instance where type assertion would be used over type annotation

Reply:
But I guess that explains why you have to do {} as BlahType when defaulting an object to empty in some scenarios